paimon-spark/paimon-spark-common/pom.xml

```diff
@@ -54,6 +54,12 @@ under the License.
             <version>${scala.version}</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.spark</groupId>
+            <artifactId>spark-avro_2.12</artifactId>
+            <version>${spark.version}</version>
+        </dependency>
```

Comment:
Does spark avro need to participate in compilation? If not, can you define it scope as test?

Reply:
I think it is necessary. 
You can see:https://www.databricks.com/blog/2018/11/30/apache-avro-as-a-built-in-data-source-in-apache-spark-2-4.html

Reply:
done.